odoo/models.py

```diff
@@ -4561,6 +4561,9 @@ def _generate_m2o_order_by(self, alias, order_field, query, reverse_direction, s
         # figure out the applicable order_by for the m2o
         dest_model = self.env[field.comodel_name]
         m2o_order = dest_model._order
+        if m2o_order == 'id':
+            order_direction = 'DESC' if reverse_direction else ''
+            return ['"%s"."%s" %s' % (alias, order_field, order_direction)]
```

Comment:
```suggestion
            return self._generate_order_by_inner(alias, m2o_order, query, reverse_direction)
```

Reply:
ah yes, sorry the test in test_read_group with assetQueries was catching the issue but I had not noticed it

```diff
AssertionError: 'sele[253 chars]line"."id"orderby"test_read_group_order_line"."id"' != 'sele[253 chars]line"."order_id"orderby"test_read_group_order_line"."order_id"
             # with direct return
-            GROUP BY "test_read_group_order_line"."order_id", "test_read_group_order_line"."order_id"
-            ORDER BY "test_read_group_order_line"."order_id"
             # with return of _generate_order_by_inner
+            GROUP BY "test_read_group_order_line"."order_id","test_read_group_order_line"."id"
+            ORDER BY  "test_read_group_order_line"."id"
```
I will check the double group by